main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       synth
+//  action 1:   the synth sends midi information
+synth.sendMidi();
+//  action 2:   the synth sends audio information
+synth.sendAudio();
+//  action 3:   the synth loads its default presets
+synth.loadPresets();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       muscle
+//  action 1:   the muscle contracts
+muscle.contract();
+//  action 2:   the muscle rests
+muscle.rest();    
+//  action 3:   the muscle grows
+muscle.grow();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       computer
+//  action 1:   the computer consumes electricity
+computer.consumeElectricity();
+//  action 2:   the computer stores data
+computer.memory();
+//  action 3:   the computer regulates temperature
+computer.regulateTemp();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       monkey
+//  action 1:   the monkey climbs a tree
+monkey.climbTree();
+//  action 2:   the monkey peels fruit
+monkey.peelFruit();
+//  action 3:   the monkey grooms itself
+monkey.groom();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       aircraft
+//  action 1:   the aircraft holds cargo
+aircraft.holdCargo();
+//  action 2:   the aircraft glides
+aircraft.glide();
+//  action 3:   the aircraft burns fuel
+aircraft.burnFuel();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       alarm
+//  action 1:   the alarm checks the time
+alarm.checkTime();
+//  action 2:   the alarm plays an alert sound
+alarm.alert();
+//  action 3:   the alarm increases in volume if not responded to
+alarm.alertLouder();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       plant
+//  action 1:   the plant synthesises light
+plant.synthesiseLight();
+//  action 2:   the plant grows flowers
+plant.growFlower();
+//  action 3:   the plant stores water
+plant.storeWater();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       mobile phone
+//  action 1:   the phone searches for a signal provider
+phone.signalSearch();
+//  action 2:   the phone notifies the user when battery is low
+phone.lowBatteryNotification();
+//  action 3:   the phone charges it battery
+phone.chargeBattery();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.